### PR TITLE
Coin: Added additional trigger

### LINKED ISSUE #2646: Coin: Doesn't trigger on coin flip
@mattlehning 

------
IA Page:  http://duck.co/ia/view/coin

### PR DESCRIPTION
**Pull request by mr-karan**
Fixes #2646 . Added additional trigger `coin flip` as mentioned in the issue

Link to IA : https://duck.co/ia/view/coin

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mr-karan Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Coin.pm [hunk 1]
```diff
@@ -6,7 +6,7 @@ use DDG::Goodie;
 
 zci is_cached => 0;
 
-triggers start => 'flip', 'toss', 'coin', 'heads';
+triggers start => 'flip', 'toss', 'coin', 'heads' ,'coin flip';
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Coin.pm]: I think we can just handle this below in the handler - `coin` will already trigger it on this.

Also, we should get some new tests to make sure the new trigger works.

**Comment by moollaza** [comment]
@mr-karan thanks again for all the hard work you've put into improving this Coin Goodie. You've done a great job so far! We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

It looks like this is almost done?  If you're stuck or need any help, I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

**Comment by GuiltyDolphin** [comment]
@mr-karan This is about ready to go - just need to add a test or two to make sure it triggers!

**Comment by mr-karan** [comment]
@GuiltyDolphin Is it fine now ?

**Comment by mr-karan** [comment]
I'm unable to debug as to why it shows no result. I also tried using `startend`

File: t/Coin.t [hunk 2]
```diff
@@ -90,6 +90,14 @@ ddg_goodie_test(
             result    => qr/(heads|tails),? /
         }
     ),
+        'coin flip' => test_zci(
```
Comment (GuiltyDolphin) [inline t/Coin.t]: Lets align this with the others.

File: lib/DDG/Goodie/Coin.pm [hunk 3]
```diff
@@ -6,7 +6,7 @@ use DDG::Goodie;
...
 
 handle query_lc => sub {
     my $flips;
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Coin.pm]: We need to update the regex to allow for `coin flip`.

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Pull request closed without merging**